hack/generate.sh

```diff
@@ -18,4 +18,4 @@ set -e
 
 echo "> Generate"
 
-GO111MODULE=on go generate -mod=vendor $@
+GO111MODULE=off go generate -mod=vendor $@
```

Comment:
Hmm, I guess you idea with the explicit `GO111MODULE=off` is for the usage of this script in the extensions repositories? If yes, does it make sense again to add more details about why it is explicitly set to `GO111MODULE=off`?

```suggestion
# We need to explicitly pass GO111MODULE=off to k8s.io/code-generator as it is significantly slower
# when it is executed without GO111MODULE=off - ref https://github.com/kubernetes/code-generator/issues/100.
GO111MODULE=off go generate -mod=vendor $@
```

Reply:
Done, and added the same comment to `generate-parallel.sh` as well. Now I think every usage of `GO111MODULE=off` is commented.